tools/integration_tests/managed_folders/admin_permissions_test.go

```diff
@@ -33,18 +35,163 @@ import (
 // Boilerplate
 // //////////////////////////////////////////////////////////////////////
 
+const (
+	CreateTestFile = "createTestFile"
+)
+
 var (
-	bucket  string
-	testDir string
+	bucket           string
+	testDir          string
+	serviceAccount   string
+	localKeyFilePath string
 )
 
 type managedFoldersAdminPermission struct {
```

Comment:
struct name is misleading as there are tests for different combination of permissions